src/util/cast.js

```diff
@@ -30,6 +30,7 @@ class Cast {
     static toNumber (value) {
         // If value is already a number we don't need to coerce it with
         // Number().
+
```

Comment:
The comment is referring to the code below it, so while this is just a really tiny stylistic nitpick, I'm not sure if this newline should be here?